xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
excuse me if i seem silly but i still don't get the function of `--interactive` 
when i run in **bash**:
 `xonsh --interactive tests/sample.xsh` 
i expect it to run the script and then get into 'interactive' mode with the xonsh prompt and repl
isnt that the use of `--interactive` ?